source/features/parse-backticks.tsx

```diff
@@ -52,14 +33,6 @@ function init(): void {
 			parseBackticks(element);
 		},
 	});
-
-	// `isRepoSearch` might highlight keywords inside backticks, breaking the regular dom-formatter #3509
```

Comment:
> I see you removed some code for the repo search. Please provide a real URL to make sure the issue does not happen.

👍 Checked and added link to first post. GitHub does it natively now https://github.com/sindresorhus/refined-github/search?q=latest+reliable+button+is%3Aissue&type=Issues

<img width="1067" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/134795526-9ee77106-5172-45bb-8b4c-ec9d3be7316f.png">